eth2util/keystore/keystore.go

```diff
@@ -52,11 +52,17 @@ func StoreKeysInsecure(secrets []tbls.PrivateKey, dir string, _ confirmInsecure)
 
 // StoreKeys stores the secrets in dir/keystore-%d.json EIP 2335 Keystore files
 // with new random passwords stored in dir/Keystore-%d.txt.
+//
+// Note it doesn't ensure the folder dir exists.
```

Comment:
It does, right? when it calls `checkDir` in `storeKeysInternal` ?

Reply:
"ensure" doesn't mean "check", it means "create". It doesn't create the folder if it doesn't exist.

Reply:
cool